opsdroid/connector/rocketchat/__init__.py

```diff
@@ -46,8 +46,8 @@ def __init__(self, config, opsdroid=None):
         except (KeyError, AttributeError):
             _LOGGER.error(
                 _(
-                    "Unable to login: Access token is missing. "
```

Comment:
The space needs to be here since the  log will be logged as a single sentence/line. We simply wrote it like that to deal with linting issues when a line was longer than 79 characters. Now after we started using black, there are a lot of logging messages that got split up and should probably stay on the same line

Reply:
What's the rule around having a space after the period?   Should all sentences ending with a period have a space?

Reply:
The period should be the last character. The problem here is that the string is made up of two strings on separate lines. This is a legacy thing that we don't need to do any more. If you put them together it will make more sense.

```python
"Unable to login: Access token is missing. Rocket.Chat connector will not be available."
```

Reply:
Ran across a couple situations and not sure how we want to handle them.  

Example:  
_LOGGER.debug(payload)

There is no "text" of opsdroid in there.   Just the output of the error's payload.   

This was the idea behind the line of:

* When inserting the entire error output as the last part of the log message, do not end it with a period.

Just leave it or which way to address this?   Do we want to have _LOGGER.debug(payload, ".") 


Also.. 
  _LOGGER.debug(_("Unable to send image - " "Status Code %s."), resp.status)

Is this done to make translating easier?   Or should this be one string?

Reply:
If we are passing the whole payload i'd say don't add the punctuation since this log will be mostly tech and we might wonder why the dot is there.

`_LOGGER.debug(_("Unable to send image - " "Status Code %s."), resp.status)`

This is another example of legacy logging, this two strings should be together like this:

`_LOGGER.debug(_("Unable to send image - Status Code %s."), resp.status)`